azure/test.yml

```diff
@@ -34,23 +34,23 @@ jobs:
     parameters:
       rust: ${{ parameters.rust }}
       setup: ${{ parameters.setup }}
-  - ${{ if and(eq(parameters.single_threaded, 'true'), eq(parameters.test_ignored, 'true')) }}:
-    - script: cargo test --all --features "${{ parameters.features }}" -- --ignored --test-threads=1
-      displayName: Run tests (single-threaded w/ignored)
+  - ${{ if ne(parameters.single_threaded, 'true') }}:
+    - script: cargo test --features "${{ parameters.features }}" --all
```

Comment:
Super minor nit, but can we place `--all` before `--features` so that it matches what we do below?

Reply:
Also fixed.